src/main/java/org/candlepin/auth/CloudRegistrationAuth.java

```diff
@@ -224,31 +232,30 @@ private UserPrincipal createCloudUserPrincipal(String subject, String ownerKey)
      * valid
      *
      * @param principal
-     *  the principal for which to generate the token
+     *     the principal for which to generate the token
```

Comment:
So, this additional spacing is being introduced by Spotless apply, and so we may need to adjust the Spotless configuration for this offset space count!

Reply:
That was my guess, since a number of these were updated on methods you weren't otherwise touching or likely even looking at. And yeah, if that's the case, our spotless config needs an update.